File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops {

std::string convert(int n) {
    // Declare a vector of strings to store the results 
    std::string result; 
  
    if (n % 3 == 0) { 
        result += "Pling";
    } 
    if (n % 5 == 0) { 
        result += "Plang";
    } 
    if (n % 7 == 0) { 
        result += "Plong";
    } 

    if ( result.empty() ) {
        return std::to_string(n);
    }
    // Return the result vector 
    return result; 
} 
  
}  // namespace raindrops

File: solutions/javascript/raindrops/1/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (n) => {
  let words = [];
  if ( n % 3 === 0 ) words.push('Pling');
  if ( n % 5 === 0 ) words.push('Plang');
  if ( n % 7 === 0 ) words.push('Plong');
  if ( words.length === 0 ) words.push(n.toString());
  return words.join('');
};
